spec/serializers/evss_claim_detail_serializer_spec.rb

```diff
@@ -33,6 +33,15 @@
     end
   end
 
+  context 'with HTML in the VA representative field' do
+    let(:evss_claim) do
```

Comment:
This `let` is a lot of extra code for no real reason (cut and paste from below?)

I'll inline what i mean below

Reply:
Added comment below re: why I disagree